types/react-navigation/index.d.ts

```diff
@@ -262,7 +263,7 @@ export type NavigationStackScreenOptions = NavigationScreenOptions & {
   headerTitleStyle?: Style,
   headerTintColor?: string,
   headerLeft?: React.ReactElement<any>,
-  headerBackTitle?: string,
+  headerBackTitle?: string | null,
```

Comment:
In strict mode，It requires that declare null when value is optional.

a number of cases In `react-native.d.ts`.